disposable_cache.py

```
@@ -0,0 +1,44 @@
+CACHE_TTL = 24 * 60 * 60  # 1 day
+
+
+def _refresh_cache() -> None:
+    domains: Set[str] = set()
+    for url in BLACKLIST_URLS:
+        try:
+            resp = requests.get(url, timeout=10)
+            if resp.status_code == 200:
+                domains.update(line.strip().lower() for line in resp.text.split('\n') if line)
+        except Exception:
+            continue
```

Ruff (0.12.2)

`try`-`except`-`continue` detected, consider logging the exception

(S112)